content/docs/main/integrations/service-entry.md

```diff
@@ -0,0 +1,339 @@
+---
```

Comment:
The integration file should be under Istio. I would create two versions of this doc (one for ambient Waypoints using SE, one for an egress gateway using SE)